src/test/groovy/com/github/leonard84/techpoker/ApplicationIT.groovy

```diff
@@ -1,10 +1,12 @@
 package com.github.leonard84.techpoker
 
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.boot.web.server.LocalServerPort
+import org.springframework.boot.test.web.server.LocalServerPort
+import org.springframework.test.context.ContextConfiguration
 
 import geb.spock.GebReportingSpec
 
+@ContextConfiguration
```

Comment:
If you upgrade to `spock-spring:2.4-M1` this workaround isn't necessary anymore.

Reply:
Here I have to go again, a first 5-minute attempt didn't work. But I don't think that was due to the dependency.